examples/snippets/modules/ROOT/examples/akkastreams-grpc-java/src/main/java/sensordata/Logger.java

```diff
@@ -8,13 +8,15 @@
 import cloudflow.streamlets.StreamletShape;
 import cloudflow.streamlets.proto.javadsl.ProtoInlet;
 
+import scala.None;
```

Comment:
is not used?

Reply:
yes Ray, I missed that to optimize imports. Pushed a new update.